readthedocs/index.rst

```diff
@@ -74,14 +74,14 @@ Below is a small example of using matchms to calculate the Cosine score between
 
     # Print the calculated scores for each spectrum pair
     for score in scores:
-        (reference, query, score, n_matching) = score
+        (reference, query, score) = score
         # Ignore scores between same spectrum and
         # pairs which have less than 20 peaks in common
-        if reference is not query and n_matching >= 20:
+        if reference is not query and score['matches'] >= 20:
             print(f"Reference scan id: {reference.metadata['scans']}")
             print(f"Query scan id: {query.metadata['scans']}")
-            print(f"Score: {score:.4f}")
-            print(f"Number of matching peaks: {n_matching}")
+            print(f"Score: {score['score']:.4f}")
```

Comment:
https://github.com/matchms/matchms/blob/add_querying_options/README.rst#example should also use `score['score']` instead tuple splat.

Reply:
Good catch. I changed the example.